internal/pkg/aws/codepipeline/codepipeline.go

```diff
@@ -162,13 +162,21 @@ func (c *CodePipeline) getStage(s *cp.StageDeclaration) (*Stage, error) {
 		switch category {
 
 		case "Source":
```

Comment:
Hmm this is interesting the `internal/aws/` pkgs are aimed to not contain any copilot specific concepts and the higher up packages should ideally do this sort of transformation 
(an example of this done nicely is the `internal/aws/ecs` and `ecs` pkgs. The `ecs` pkg depends on `internal/aws/ecs` and all other packages depend on `ecs`)

We don't need to do this now but let's keep it in mind!